cmake/onnxruntime_hosting.cmake

```diff
@@ -39,6 +39,7 @@ endif()
 file(GLOB_RECURSE onnxruntime_hosting_srcs
   "${ONNXRUNTIME_ROOT}/hosting/main.cc"
   "${ONNXRUNTIME_ROOT}/hosting/environment.cc"
+  "${ONNXRUNTIME_ROOT}/hosting/request_handler.cc"
```

Comment:
If this is a request handler specific for http request, I would prefer to have it under http folder. Also, we may end up having multiple handlers, so you may want to name it something like `predict_request_handler.cc`? Other cases could be `status_request_handler.cc` and `metadata_request_handler.cc`

Reply:
Done and done.